enterprise/admin/views.py

```diff
@@ -311,6 +318,15 @@ def _enroll_users(cls, enterprise_customer, emails, course_id, mode, request, no
         enrollment_client = EnrollmentApiClient()
         course_details = CourseCatalogApiClient(request.user).get_course_run(course_id)
 
+        if not course_details:
+            logging.warning(
+                _(
+                    "Course details were not found for course key {}. "
```

Comment:
I think we should be more specific here -- can we say that course details for the course key were not returned by the Catalog Service